md5.js

```diff
@@ -148,8 +148,8 @@
   md5._digestsize = 16;
 
   module.exports = function (message, options) {
-    if(typeof message == 'undefined')
-      return;
+    if (message === 'undefined' || message === null)

```

Comment:
I just noticed that `'undefined'` should not be a string here. The test still passes however because an error gets thrown further down the stack, here: https://github.com/pvorb/node-md5/blob/master/md5.js#L10.

`TypeError: Cannot read property 'constructor' of undefined`